src/test/java/it/gov/pagopa/payhub/activities/mapper/ingestionflow/debtpositiontypeorg/DebtPositionTypeOrgMapperTest.java

```diff
@@ -27,12 +37,23 @@ void givenDebtPositionTypeIngestionFlowFileWithPodamFactoryWhenMapThenOk() {
     Long debtPositionTypeId = 123L;
     Long organizationId = 456L;
 
+    OrgSilService orgSilServicePaidNotificationOutcome = new OrgSilService();
+    orgSilServicePaidNotificationOutcome.setOrgSilServiceId(111L);
+    OrgSilService orgSilServiceActualization = new OrgSilService();
+    orgSilServiceActualization.setOrgSilServiceId(222L);
+    when(orgSilServiceServiceMock.getAllByOrganizationIdAndServiceType(organizationId, OrgSilServiceType.PAID_NOTIFICATION_OUTCOME))
+        .thenReturn(Collections.singletonList(orgSilServicePaidNotificationOutcome));
+    when(orgSilServiceServiceMock.getAllByOrganizationIdAndServiceType(organizationId, OrgSilServiceType.ACTUALIZATION))
+        .thenReturn(Collections.singletonList(orgSilServiceActualization));
+
     var result = debtPositionTypeMapper.map(dto, debtPositionTypeId, organizationId);
 
     Assertions.assertNotNull(result);
+    Assertions.assertEquals(111L, result.getNotifyOutcomePushOrgSilServiceId());
+    Assertions.assertEquals(222L, result.getAmountActualizationOrgSilServiceId());
     checkNotNullFields(result, "creationDate", "updateDate", "updateOperatorExternalId",
-        "updateTraceId", "debtPositionTypeOrgId","serviceId","notifyOutcomePushOrgSilServiceId","amountActualizationOrgSilServiceId");
-    //todo : remove exclusion in checkNotNullFields for serviceId, notifyOutcomePushOrgSilServiceId e amountActualizationOrgSilServiceId with task https://pagopa.atlassian.net/browse/P4ADEV-3224
+        "updateTraceId", "debtPositionTypeOrgId", "serviceId");
+    // ora i campi notifyOutcomePushOrgSilServiceId e amountActualizationOrgSilServiceId sono valorizzati
```

Comment:
please, remove this italian comment (not usefull to translate it into english)

os: why should serviceId null? this is the IO serviceId... why are we not accepting it?